src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAGIC_NUMBER = 10;
```

Comment:
one of the goals of extracting a magic number to a constant is that the constant can have **meaningful** naming and it's easy to understand how it is used. Change constant naming to mirror its purpose, like DEFAULT_RADIUS